lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -219,7 +219,7 @@ def add_resources_bundle_targets
           def create_xcconfig_file
             path = target.xcconfig_path
             xcconfig_gen = Generator::XCConfig::PodXCConfig.new(target)
-            xcconfig_gen.save_as(path)
+            update_changed_file(xcconfig_gen, path)            
```

Comment:
nit extra space after this line, please delete or run `rubocop -a` I think it will take care of it.